### PR TITLE
Upgrade from Chromium 88 to Chromium 89 (1.21.x).

### PR DESCRIPTION
**Pull request by mkarolin**
Fixes brave/brave-browser#13238
Fixes brave/brave-browser#14065
Fixes brave/brave-browser#14112
Fixes brave/brave-browser#14165
Fixes brave/brave-browser#14167
Fixes brave/brave-browser#14227
Fixes brave/brave-browser#14235
Fixes brave/brave-browser#14284

Related PR: brave/brave-browser#14255 

Uplift from #7460
Uplift from #7900 
Uplift from #7936 
Uplift from #7975 
Uplift from #7988 
Uplift from #8000
Uplift from #8027 

## Submitter Checklist:

- [ ] There is a [ticket](https://github.com/brave/brave-browser/issues) for my issue
- [ ] Used Github [auto-closing keywords](https://docs.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue) in the PR description above
- [ ] Wrote a good [PR/commit description](https://google.github.io/eng-practices/review/developer/cl-descriptions.html)
- [ ] Added appropriate labels (`QA/Yes` or `QA/No`; `release-notes/include` or `release-notes/exclude`; `OS/...`) to the associated issue
- [ ] Checked the PR locally: `npm run test -- brave_browser_tests`, `npm run test -- brave_unit_tests`, `npm run lint`, `npm run gn_check`, `npm run tslint`
- [ ] Ran `git rebase master` (if needed)
- [ ] Requested a security/privacy review as needed

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header
- [ ] Adequate test coverage exists to prevent regressions
- [ ] Major classes, functions and non-trivial code blocks are well-commented
- [ ] Changes in component dependencies are properly reflected in `gn`
- [ ] Code follows the [style guide](https://chromium.googlesource.com/chromium/src/+/HEAD/styleguide/c++/c++.md)
- [ ] Test plan is specified in PR before merging

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on
- [ ] All relevant documentation has been updated, for instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-Compatibility-Exceptions-in-Brave
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide
  - [ ] https://github.com/brave/brave-browser/wiki/P3A

### CODE REVIEW

File: app/settings_brave_strings.grdp
Comment (AlexeyBarabash) [inline app/settings_brave_strings.grdp:167]: Seems Chromium leftover, `Chromebook` => `Bravebook`

**Review by AlexeyBarabash — APPROVED** [review]
sync changes look good

**Review by simonhong — APPROVED** [review]

**Review by kjozwiak — APPROVED** [review]
Uplift into `1.21.x` approved after deliberating with @brave/uplift-approvers. QA went through the manual passes as per https://github.com/brave/brave-browser/issues/13968 & https://github.com/brave/qa-resources/issues/207.

**Pull request merged**
